naloge/pot.tex

```diff
@@ -19,6 +19,72 @@
 pri vozliščih pa stroški prenočitve v kraju.
 \end{enumerate}
 \end{vprasanje}
+
+
+
+
 \begin{odgovor}
+\begin{enumerate}[(a)]
+\item Imamo neusmerjen utežen graf. Problem bi bilo najlažje reševati s prilagojenim Dijkstrovim algoritmom, kjer bi dodali še uteži na vozliščih.
+
+\item Potek samega reševanja problema je prikazan v tabeli ~\tab. Cena poti od LJ do BX je 150. Pot pa je sledeča: LJ - WI - BN - LU - BX.
+\end{enumerate}
+\begin{tabela}
+$$
+\begin{array}{c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c|c}
+u & v & Q & d[BX], o[BX] &   d[AM], o[AM] &  d[BL], o[BL] &  d[PR], o[PR] &  d[BS], o[BS] &  d[PA], o[PA] & d[LU], o[LU] &  d[BN], o[BN] & d[WI], o[WI] &  d[BU], o[BU] &  d[MO], o[MO] &  d[VE], o[VE] &  d[LJ], o[LJ] &  d[ZG], o[ZG] \\ \hline
```

Comment:
Svetujem, da poskusiš tabelo nekoliko skrčiti tako, da v glavi navajaš samo imena vozlišč (daj jih v `\text{}`, da ne bodo poševna in bodo lepše izgledala), v vsakem od teh stolpcev pa navajaš številsko vrednost s predhodnikom v indeksu (npr. `35_{\text{LJ}}`). Tabelo lahko zmanjšaš tudi tako, da postaviš `\begin{small} ... \end{small}` okoli `$$ $$`.